From 6fbcadb45f9f0fd90cddc0fcdfa13de4bb0b83c9 Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Wed, 9 Aug 2017 15:44:14 +0200
Subject: [PATCH] Break cycle dependency with reno

openstackdocstheme requires reno for testing, but reno requires
openstackdocstheme. To break this, make reno an optional test
requirement.

Change-Id: I267e0216089038a15df8221efc1fcd365e0809be
---
 setup.cfg             | 2 ++
 test-requirements.txt | 2 --
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: openstackdocstheme-1.16.1/setup.cfg
===================================================================
--- openstackdocstheme-1.16.1.orig/setup.cfg
+++ openstackdocstheme-1.16.1/setup.cfg
@@ -24,6 +24,8 @@ test =
     # os-api-ref needs openstackdocstheme which needs os-api-ref (cycle dep)
     # so put os-api-ref in extras to make it an optional test dependency
     os-api-ref>=1.0.0 # Apache-2.0
+    # reno needs openstackdocstheme which needs reno (cycle dep)
+    reno!=2.3.1,>=1.8.0 # Apache-2.0
 
 [global]
 setup-hooks = 
Index: openstackdocstheme-1.16.1/test-requirements.txt
===================================================================
--- openstackdocstheme-1.16.1.orig/test-requirements.txt
+++ openstackdocstheme-1.16.1/test-requirements.txt
@@ -6,5 +6,3 @@ hacking!=0.13.0,<0.14,>=0.12.0 # Apache-
 
 # this is required for the docs build jobs
 sphinx>=1.6.2 # BSD
-
-reno!=2.3.1,>=1.8.0 # Apache-2.0
